### PR TITLE
Reenable github actions and fix size spec bug

### PR DESCRIPTION
**Pull request by stevenmburns**
@ekiwi Thanks for adding the formal checks. I'll look at them and try to get the working locally as well as in github actions.

### CODE REVIEW

**Comment by stevenmburns** [comment]
@ekiwi Is this a reasonable way to do a formal equivalence check between two implementation of a simple ALU?

File: src/test/scala/alu/ALUSpecFormal.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import chiseltest._
+import chiseltest.formal._
+import org.scalatest.freespec.AnyFreeSpec
+import chisel3.experimental.BundleLiterals._
+
+import testutil._
+
+class AluMiterWithChecker(factory0 : () => AluIfc, factory1 : () => AluIfc) extends AluMiter(factory0, factory1) {
+  assert(same || (mode =/= 1.U && mode =/= 2.U && mode =/= 4.U))
+}
```
Comment (stevenmburns) [inline src/test/scala/alu/ALUSpecFormal.scala:14]: @ekiwi Is there a good way to specify restrictions on inputs?

Comment (ekiwi) [reply src/test/scala/alu/ALUSpecFormal.scala:14]: Yes. You can use an `assume( ... some boolean expression ...)`

Comment (ekiwi) [reply src/test/scala/alu/ALUSpecFormal.scala:14]: So I guess something like:
```
assume(mode === 1.U || mode === 2.U ||  mode === 4.U)
assert(same)
```

**Pull request merged**